docs/configure-archive-logs.md

```diff
@@ -1,6 +1,7 @@
 # Configuring Archive Logs
 
-⚠️ We do not recommend you rely on Argo Workflows to archive logs. Instead, use a conventional Kubernetes logging facility.
+!!! Warning "Not recommended"
+    We [do not recommend](#why-doesnt-argo-workflows-recommend-using-this-feature) you rely on Argo Workflows to archive logs. Instead, use a dedicated Kubernetes capable logging facility.
```

Comment:
```suggestion
    We do not recommend relying on Argo to archive logs as it is naive and not purpose-built for indexing, searching, and storing logs.
    This feature is provided as a convenience to quickly view logs of garbage collected Pods in the Argo UI, but we [recommend](#suggested-alternatives) you integrate a dedicated, Kubernetes-aware logging facility.
```

- 1 sentence per line, per [style guide](https://argo-workflows.readthedocs.io/en/release-3.5/doc-changes/)
- Generally the docs refer to Workflows as "Argo"; this seems to be an outlier. I think that is simpler for a self-reference as well
- As mentioned in my other comment, state the reason why they shouldn't use this explicitly in the admonition. Link to the new section title after.
- Also mention why this feature even exists if we recommend something else.
- "Kubernetes-aware" is the general term as opposed to "Kubernetes capable"